pyglui/menus.pxi

```diff
@@ -582,21 +586,22 @@ cdef class Scrolling_Menu(Base_Menu):
             #mouse:
             self.scrollbar.handle_input(new_input,visible)
             #scrollwheel:
-            if new_input.s.y and visible and self.element_space.mouse_over(new_input.m):
+            if new_input.s.y and visible and self.element_space.mouse_over(new_input.mouse_pos):
```

Comment:
while more legible, we want this to be fast. Can you remove the mouse_pos property and use m OR rename m to mouse_pos everywhere?